core/workspace_comment_svg.js

```diff
@@ -163,6 +163,10 @@ Blockly.WorkspaceCommentSvg.prototype.initSvg = function() {
   if (!this.getSvgRoot().parentNode) {
     this.workspace.getBubbleCanvas().appendChild(this.getSvgRoot());
   }
+
+  if (this.textarea_) {
+    this.textarea_.select();
+  }
```

Comment:
I think this is on the right track! 

My only concern is that this `initSvg` method will get called in [fromXml](https://github.com/google/blockly/blob/11987dc6d6aa16138b36237e8959f2a3f91eb4e2/core/workspace_comment_svg.js#L584) which will cause all of the comments to be selected when someone loads a workspace from xml. This can be tested by going to the [playground](https://blockly-demo.appspot.com/static/tests/playground.html), creating a workspace comment, changing the text, and then hitting the "Export to XML" and the "Import from XML" buttons.

I think this can be fixed if we add an optional parameter to this method that can be passed in as true in `fromXml`. We can then check that the optional parameter is false before selecting the area.

Thanks! And let me know if you have any questions!

Reply:
Thanks for the help! 

I am not sure about the final status: :

I created a new boolean parameter `deSelectText` if it is false I am selecting the text
From `fromXml` method I set the flag to true. 

It feels a bit strange to me, especially with the variable name, let me know what you think! 

again thanks for the help!